src/current/v23.1/sso-db-console.md

```diff
@@ -6,105 +6,110 @@ docs_area: manage
-	- `^([^@]+)@[^@]+$` matches any email address (defined as a string containing one `@` sign) and extracts a username from the string to the left of `@`.
-	- `^(.+)$` maps the claim directly to a principal.
-- `autologin` is a Boolean. When `true`, upon loading the DB Console login page, the browser will automatically initiate the OIDC authentication process by redirecting to `/oidc/v1/login` instead of waiting for the user to log in manually or click the OIDC login button.
-
-## Example (Google OAuth 2.0)
-
-These steps demonstrate how to enable SSO authentication for the DB Console on a [local secure cluster]({% link {{ page.version.version }}/secure-a-cluster.md %}) using [Google's OAuth 2.0](https://developers.google.com/identity/protocols/oauth2) implementation.
+
+## Log in to a cluster's DB Console with SSO
+
+From the user's perspective, once the cluster is properly configured to an identity provider, the sign-in flow is as follows:
+
+1. A user opens the cluster's DB Console, and clicks on the **Log in with your OIDC provider** button that renders on the page.
+1. The user is redirected to an external identity provider.
+1. The user authenticates successfully with the provider, which completes the OAuth flow.
+1. The user is redirected to the CockroachDB cluster.
+1. CockroachDB creates a web session for the SQL user in a new browser tab.
+1. In the original browser tab, the user is redirected to the [DB Console Cluster Overview]({% link {{ page.version.version }}/ui-cluster-overview-page.md %}).
+
+## Provision an OAuth client using Google Cloud Platform (GCP)
+
+These steps demonstrate how to create an OIDC auth client in Google Cloud Platform to use for SSO authentication to the DB Console.
 
 1. Open the [Credentials page](https://console.developers.google.com/apis/credentials) for your account at Google APIs.
 
 1. Click **+ CREATE CREDENTIALS** and select **OAuth client ID**. Specify a **web application** from the pulldown menu.
 
-1. Note the *client ID* and *client secret* of the OAuth 2.0 client. You can find these in the dialog that appears after you create the client, or in the details view for the client:
+1. Note the *client ID* and *client secret* of the OAuth 2.0 client—you will need to configure your cluster to use these values.
+
+1. Add your cluster's callback URL to the list of **Authorized redirect URIs**. On a local cluster, this will be `https://{ your cluster's domain }:8080/oidc/v1/callback`. Subsequently, when configuring your cluster, you will need to ensure that the cluster setting `server.oidc_authentication.redirect_url` has the same value.
+
+	- For a {{ site.data.products.core }} cluster, The domain is `localhost`. 
+	- For a {{ site.data.products.dedicated }} cluster, find the domain by opening the DB Console from your cluster's **Tools** tab in DB Console.
```

Comment:
```suggestion
1. Add your cluster's callback URL to the list of **Authorized redirect URIs**. On a local cluster, this will be `https://{ your cluster's domain }:8080/oidc/v1/callback`. Subsequently, when configuring your cluster, you will need to ensure that the cluster setting `server.oidc_authentication.redirect_url` has the same value.
	- For a {{ site.data.products.core }} cluster, The domain is `localhost`. 
	- For a {{ site.data.products.dedicated }} cluster, find the domain by opening the DB Console from your cluster's **Tools** tab in DB Console.
```